recipes/tinytimer/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: tinytimer
+  version: "0.0.0"
+
+source:
+  fn: tinytimer-0.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/5c/87/75d3fbe15aabd45d9b70702241787cf1f7f30dd9fabcd9bc89d828c7661d/tinytimer-0.0.0.tar.gz
+  md5: 83d6988d1ca1702fcbd9ba5eae9d4fd7
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Should be platform independent, so don't skip. Also consider making `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages